test_notification.py

```diff
@@ -0,0 +1,187 @@
+#! /usr/bin/env python
```

Comment:
Thanks. It turned out well after solving the permissions puzzle and the quoting puzzle through 2 shells + adb. Two Stack Overflow links in the comments.